homeassistant/components/deconz/__init__.py

```diff
@@ -76,3 +78,18 @@
     options = {**old_options, **new_options}
 
     hass.config_entries.async_update_entry(config_entry, options=options)
+
+
+async def async_add_uuid_to_config_entry(hass, config_entry):
+    """Add UUID to config entry to help discovery identify entries."""
+    if CONF_UUID in config_entry.data:
+        return
+
+    gateway = get_gateway_from_config_entry(hass, config_entry)
+
+    old_config = dict(config_entry.data)
+    new_config = {CONF_UUID: gateway.api.config.uuid}
+
+    config = {**old_config, **new_config}
```

Comment:
FYI this also is possible. Also no need to copy `old_config` if you are going to then `**` it.

```suggestion
    config = {**old_config, CONF_UUID: gateway.api.config.uuid}
```